lib/generate-icon-set-from-css.js

```diff
@@ -6,7 +6,7 @@ var _ = require('lodash');
 var fs = require('fs');
 
 function extractGlyphMapFromCss(files, selectorPattern) {
-  var styleRulePattern = '(\\.[A-Za-z0-9_.:, \\n\\t-]+)\\{[^}]*content: ?["\\\']\\\\([A-Fa-f0-9]+)["\\\'][^}]*\\}';
+  var styleRulePattern = '(\\.[A-Za-z0-9_.:, \\n\\t-]+)\\{[^}]*content: ?["\\\'](?:\\\\([A-Fa-f0-9]+)|([^"\\\']+))["\\\'][^}]*\\}';
```

Comment:
Without the `+` the regex doesn't match some characters (I think they are multibyte characters). I think the char 🔎  in the example is one of them.
That's also why I put the actual character in result when length > 1: `(actualChar.length > 1 ? actualChar : actualChar.charCodeAt())`. I tried to use codePointAt() instead of charCodeAt() and I think it was outputting the correct char code, but these chars weren't displaying properly in RN.